build/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN dnf install -y golang-$(sed -En 's/^go +(.*+)$/\1/p' go.mod).*
 
 COPY . .
 
-RUN --mount=type=cache,target=/root/.cache/go-build GOOS=linux GOARCH=${TARGETARCH} go build -o /manager ./cmd/handler
+RUN GOOS=linux GOARCH=${TARGETARCH} go build -o /manager ./cmd/handler
```

Comment:
Can we put this version at Dockerfile.podman ? the cache is quite convenient since it will not compile the vendoring everytime you change a line of code.